rfcs/0000-template.md

```diff
@@ -0,0 +1,46 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- Reference Issues: (fill in existing related issues, if any)
+- Implementation PR: (leave this empty)
+
+# Summary
+
+Brief explanation of the feature.
```

Comment:
I suggest to wrap these "what to do" text in `<!--` comments so contributors don't forget to remove them or at least they will not get rendered in the readable version

Reply:
It's copied from Vue's RFC. I don't have an opinion on this.

Reply:
I have also to say that this is a _nitpick_
Theoretically we can review in every single PR that these phrases should not be copied over and over again but removed/replaced by the actual content.